webpack.common.js

```diff
@@ -28,4 +29,17 @@ module.exports = env => ({
     publicPath: '/dist/',
     filename: 'bundle.js',
   },
-});
+  plugins: [
+    new webpack.DefinePlugin({
+      FIREBASE_CONFIG: {
```

Comment:
FIREBASE_CONFIG will become a global value available in our app that contains all of the required config values for connecting to firebase, which are in turn grabbed from our build environment's env vars (in node, `process.env` allows us to grab the current build environment's env vars).